docs/src/main/asciidoc/_developer-guide/ejb3/jboss-ejb3.xml_Reference.adoc

```diff
@@ -78,7 +78,7 @@ or to all beans in the deployment by using `*` as the `ejb-name`.
 ==== The security namespace urn:security
 
 This allows you to set the security domain and the run-as principal for
-an Jakarta Enterprise Beans.
+a Jakarta Enterprise Beans.
```

Comment:
probably "Bean", unless this is about something that's scoped to > 1 bean, in which case no "an" or "a"